tests/system/requests/test_download.py

```diff
@@ -63,19 +63,20 @@
     }, {
         u'path': os.path.realpath(os.path.join(DATA_DIR, u'gzipped.txt.gz')),
         u'uncompressed':
-            os.path.realpath(os.path.join(DATA_DIR, u'gzipped.txt')),
+            os.path.realpath(os.path.join(DATA_DIR, u'gzipped.txt.gz')),
```

Comment:
This change defeats the purpose of the `uncompressed` key, which is to point to the file in its un-gzipped form.